src/utils/Card.js

```diff
@@ -74,6 +74,16 @@ export const CARD_CATEGORY_DETECTORS = {
     details.legalities.Commander === 'legal' &&
     ((details.type.includes('Legendary') && details.type.includes('Creature')) ||
       details.oracle_text.includes('can be your commander')),
+  spell: (details) => !details.type.includes('Land') && !details.type.includes('Conspiracy'),
```

Comment:
use the special card type filter we use for the draftbots so that it also filters out planes, vanguards, tokens, and other things that don't go into decks.